external-api/api-tests/v1/party-urn.api-test.ts

```diff
@@ -1,29 +1,32 @@
 import { app } from '../../src/createApp';
 import { api } from '../api';
+import MockAdapter from 'axios-mock-adapter';
+import axios, { HttpStatusCode } from 'axios';
 
+const { APIM_MDM_URL } = process.env;
 const { get } = api(app);
 
-const mockResponse = { status: 200, data: 'mock response' };
-
-jest.mock('axios', () => jest.fn(() => Promise.resolve(mockResponse)));
+// Mock Axios
+const axiosMock = new MockAdapter(axios);
+axiosMock.onGet(`${APIM_MDM_URL}customers?partyUrn=03827491`).reply(HttpStatusCode.Ok, {});
 
 describe('/party-db/urn', () => {
   describe('GET /party-db/urn', () => {
-    it('returns a 200 response with a valid party urn', async () => {
+    it('returns a HttpStatusCode.Ok response with a valid party urn', async () => {
       const { status } = await get('/party-db/urn/03827491');
 
-      expect(status).toEqual(200);
+      expect(status).toEqual(HttpStatusCode.Ok);
     });
   });
 
   const invalidPartyUrnTestCases = [['123'], ['127.0.0.1'], ['{}'], ['[]']];
 
   describe('when party urn is invalid', () => {
-    test.each(invalidPartyUrnTestCases)('returns a 400 if you provide an invalid party urn: %s', async (partyUrn) => {
+    test.each(invalidPartyUrnTestCases)('returns a HttpStatusCode.BadRequest if you provide an invalid party urn: %s', async (partyUrn) => {
```

Comment:
LEgacy & both standards are used but `it` more often than `test` 
```suggestion
    it.each(invalidPartyUrnTestCases)('returns a 400 if you provide an invalid party urn: %s', async (partyUrn) => {
```

Reply:
Updated